testdata/output/telex__simple_telex.snap

```diff
@@ -67,4 +67,9 @@ trường
 ươi
 cười
 hường
+chửa
+chuă
+ừa
+ww
```

Comment:
This should be `w`. A `ww` in telex would be transformed as `ư` then `w`. At least that's how EVKey behaves.

Reply:
I think people needing to type a single `w` char in Vietnamese mode is kinda rare.

word | before | after
-- | -- | --
w | ww | `ww<bs>`
ww | "none" | ww
www | "none" | www
wow | wwow | `ww<bs>ow`
ư | w | w
uw | uww | uww